src/components/PhotoAnalysis/PhotoAnalysis.tsx

```diff
@@ -5,13 +5,13 @@ import {
   IdentifyLabelsOutput,
```

Comment:
We should probably take out the `eslint-disable` thing

Reply:
Sure, but then we need to maybe change a lint rule to allow for the any type and catch unnecessary usage of it in PRs

Reply:
This is fixed in #35 (with PR #32)